src/schema/v2/CollectorProfile/collectorProfile.ts

```diff
@@ -113,7 +113,23 @@ export const CollectorProfileFields: GraphQLFieldConfigMap<
   inquiryRequestsCount: {
     type: new GraphQLNonNull(GraphQLInt),
     resolve: ({ artwork_inquiry_requests_count }) =>
-      artwork_inquiry_requests_count,
+      artwork_inquiry_requests_count || 0,
+  },
+  savedArtworksCount: {
+    type: new GraphQLNonNull(GraphQLInt),
+    resolve: ({ saved_artworks_count }) => saved_artworks_count || 0,
```

Comment:
**suggestion:** I think `saved_artworks_count` (and other added resolving fields) `size` in gravity is returning 0 by default. `|| 0` is not needed here.

Reply:
I think most, if not all, are safe on the gravity side, but I was still getting `Cannot return null for non-nullable field CollectorProfileType...` before they finished resolving 🤷

Reply:
I don't know if something changed, but last week `inquiryRequestsCount` was fine, but yesterday it was complaining.

Reply:
It might be related with data-sync? Regardless, I think we might want to confirm this with the test and we can move on:)

Reply:
Avoid trusting Gravity, fallback to 0

Reply:
I haven't found any examples with fallback on the root level. Why would one avoid trusting Gravity? It should be the source of truth! 😅

Reply:
I feel like there are no many on the root level like this, maybe in resolvers more tough

Reply:
Makes sense, add more defence - wasn't ever the issue 😅 
I will not comment if we add fallbacks to the clients as well 💯

Reply:
😄 well the clients will have type information from Metaphysics